### ISSUE TITLE
Update TravisCI to deploy from the master branch

### ISSUE THREAD

**Issue by BaaniLeen** [systers/systers.github.io#295]
## Description
As a contributor,
I need to Update TravisCI to deploy from the master branch once the code is merged into the master branch.

### Update [Required]
- [ ] .travis.yml

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.75 hour = 0.5 hours + 0.25 to create a PR

@bethanyr @ancyp Can I please work on the issue?

**Comment by BaaniLeen** [comment]
Solved using PR#294
